pkgs/tools/admin/pgadmin/default.nix

```diff
@@ -1,21 +1,23 @@
-{ stdenv
-, lib
+{ lib
 , python3
 , fetchurl
 , zlib
 , mkYarnModules
 , sphinx
 , nixosTests
+, pkgs
 }:
 
 let
 
   pname = "pgadmin";
-  version = "6.7";
+  majorVersion = "6";
+  minorVersion = "8";
+  version = "${majorVersion}.${minorVersion}";
```

Comment:
Please revert that. We have ``lib.versions.majorMinor``, ``lib.versions.major`` and ``lib.versions.minor``.